### PR TITLE
Tempo: search fixes

### PR DESCRIPTION
**Pull request by zoltanbedi**
**What this PR does / why we need it**:

@wardbekker found some issues in the Tempo search feature.

Fixes issues:
- [x] Add custom width to time and trace id
- [x] Run and show query in search after page refresh
- [x] Do not show log panel when return type is log stream

### CODE REVIEW

**Comment by wardbekker** [comment]
thanks for looking into this @zoltanbedi !

File: public/app/plugins/datasource/tempo/resultTransformer.ts [hunk 1]
```diff
@@ -85,8 +91,10 @@ export function transformTraceList(
   datasourceName: string,
   traceRegexs: string[]
 ): DataQueryResponse {
-  const frame = createTableFrame(response.data[0], datasourceId, datasourceName, traceRegexs);
-  response.data[0] = frame;
+  response.data.forEach((data, index) => {
+    const frame = createTableFrame(data, datasourceId, datasourceName, traceRegexs);
```
Comment (aocenas) [inline public/app/plugins/datasource/tempo/resultTransformer.ts:95]: Which change btw prevents showing log panel? Seems like we were already returning table frame here.

File: public/app/plugins/datasource/tempo/resultTransformer.ts [hunk 2]
```diff
@@ -85,8 +91,10 @@ export function transformTraceList(
   datasourceName: string,
   traceRegexs: string[]
 ): DataQueryResponse {
-  const frame = createTableFrame(response.data[0], datasourceId, datasourceName, traceRegexs);
-  response.data[0] = frame;
+  response.data.forEach((data, index) => {
```
[the remark this answers is not in the document]
Comment (zoltanbedi) [reply public/app/plugins/datasource/tempo/resultTransformer.ts:94]: yes

File: public/app/plugins/datasource/tempo/resultTransformer.ts (hunk 1 above)
Comment (zoltanbedi) [reply public/app/plugins/datasource/tempo/resultTransformer.ts:95]: yes but only for the first frame and all the other had `preferredVisualisationType: 'logs',`

**Review by aocenas — APPROVED** [review]

**Pull request merged**